blog/2023-11-20-second-revival-milestone-unveiled/index.md

```diff
@@ -0,0 +1,105 @@
+---
+posttype: blog
+title: "Revival Milestone 2 Unveiled - Elevating Quality, Boosting Performance, Advancing to Java 17!"
+cover: "./cover.jpg"
+description: "Unveiling Terasology Revival Milestone 2 - where quality meets performance, and Java 17 takes the spotlight. We’re celebrating our achievements and planning our path ahead. Ready to join the journey? New contributors welcome!"
+author: "Niruandaleth"
+date: "2023-11-20"
+tags: ["Announcement"]
+---
+
+_Unveiling Terasology Revival Milestone 2 - where quality meets performance, and Java 17 takes the spotlight.
+We’re celebrating our achievements and planning our path ahead.
+Ready to join the journey? New contributors welcome!_
+
+Hey there, fellow gooeys! Welcome to Terasology, where blocks meet potential ✨
+Our project was born in 2011 in the desire to provide a modular canvas for diverse gameplay ideas in a dynamic voxel world with creative possibilities as endless as the blocks themselves.
+While we wear our history with pride, our codebase outgrew our maintainer capacity over the years, collecting rust and dust in the face of dwindling contributor and contribution numbers.
```

Comment:
I do think we (although maybe not all) do. I myself am not happy with everything that happened in the project's history, both before and since I joined, but I am proud that the project survived that long and I'm proud of everybody that contributed (in whatever way, extent, and quality).
I agree that this is about looking forward, not back, but it is also about our revival effort and a reader might not have the context why there is a revival in the first place, so I'd like to leave it in to provide that context.